rocksdb/rocksdb.nim

```diff
@@ -63,6 +63,42 @@ type
     writeOpts: WriteOptionsRef
     ingestOptsPtr: IngestExternalFilesOptionsPtr
 
+proc listRocksDbCFs*(
```

Comment:
we strive to use the official names for functions here.

```suggestion
proc listColumnFamilies*(
```

Reply:
ok